playground/optimize-deps/dep-with-builtin-module-cjs/index.js

```diff
@@ -1,8 +1,8 @@
 // no node: protocol intentionally
 // eslint-disable-next-line import/no-nodejs-modules
-const fs = require('fs')
+const fs = require('node:fs')
 // eslint-disable-next-line import/no-nodejs-modules
-const path = require('path')
+const path = require('node:path')
```

Comment:
These need to be reverted, `node:` is missing intentionally for the test.

Reply:
Oops, comment not seen